nixos/modules/virtualisation/docker-containers.nix

```diff
@@ -211,6 +211,10 @@ let
     after = [ "docker.service" "docker.socket" ] ++ mkAfter;
     requires = after;
 
+    preStart = concatStringsSep "\n" (["${pkgs.docker}/bin/docker rm -f ${name} || true"] ++
+                                      (optional (container.imageFile != null) "${pkgs.docker}/bin/docker load -i ${container.imageFile}"));
```

Comment:
it may be done prettier:
```nix
let # syntax highlight only

path = [ pkgs.docker ];
preStart = ''
  docker rm -rf ${name} || true
  ${optionalString (container.imageFile != null) ''
    docker load -i ${container.imageFile}
  ''}
'';
```

Reply:
Yes, that looks way cleaner. I just wanted to keep the changes minimal by only moving the values around. Do you think I should add docker to the path with this PR or is it better to do so it in a separate PR?

Reply:
I suppose the purpose of putting docker in the path here would be so any extra commands added by users of the module can just call `docker` and not need to spell out the exact store path.  Might as well go in this PR, I guess?

Reply:
That makes sense. I added and pushed it.